practice_app/practice_app/settings.py

```diff
@@ -41,6 +41,7 @@
     'rest_framework',
     'apps.advice',
     'apps.article',
+    'apps.news'
```

Comment:
Missing comma in the end causing errors.

Reply:
it was due to the inconsistency between master, sorry I did not double check the commas.